beacon_node/network/src/beacon_processor/worker/gossip_methods.rs

```diff
@@ -785,6 +785,14 @@ impl<T: BeaconChainTypes> Worker<T> {
 
                 verified_block
             }
+            Err(e @ BlockError::PublishError) | Err(e @ BlockError::SlashablePublish) => {
```

Comment:
Lets put slashable publish in its own branch and penalise the peer with a `gossip_block_low` penalty. We should log a WARN in this case too, rather than an error, as equivocations are outside our node's control and might happen from time to time.

The peer penalty covers the case where a peer is spamming us with equivocating blocks, and we want to ban them to prevent a DoS. An honest peer might forward us _one_ equivocating block, but no more than that. It takes 10 low-tolerance penalties to get banned, so spurious bans of honest peers shouldn't be an issue.